src/bin/init.rs

```diff
@@ -68,20 +68,15 @@ pub fn execute(args: &ArgMatches) -> Result<()> {
     Ok(())
 }
 
-/// Obtains author name from git config file if it can be located.
+/// Obtains author name from git config file by running the `git config` command
 fn get_author_name() -> Option<String> {
-    if let Some(home) = env::home_dir() {
-        let git_config_path = home.join(".gitconfig");
-        let content = utils::fs::file_to_string(git_config_path).unwrap();
-        let user_name = content
-            .lines()
-            .filter(|x| !x.starts_with("#"))
-            .map(|x| x.trim_left())
-            .filter(|x| x.starts_with("name"))
-            .next();
-        user_name
-            .and_then(|x| x.rsplit("=").next())
-            .map(|x| x.trim().to_owned())
+    let output = Command::new("git")
+        .args(&["config", "--get", "user.name"])
+        .output()
+        .expect("failed to get user.name from git config");
```

Comment:
Can we change this `expect()` call to something like `ok()?`? That way if there was an error executing `git` (i.e. git isn't installed) we'll just say no author was found instead of panicking.